mittab/apps/tab/api.py

```diff
@@ -0,0 +1,146 @@
+import json
+
+from mittab.apps.tab.models import *
+from mittab.libs import tab_logic
+
+
+def get_num_rounds_debated(team):
```

Comment:
Is this for standings site integration? If so, can you move it to another PR?

Reply:
#whathappenswhenImessupwithgit

Yeah absolutely